lib/msal-node/README.md

```diff
@@ -94,30 +95,6 @@ There are multiple [samples](https://github.com/AzureAD/microsoft-authentication
 - [on-behalf-of](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-samples/on-behalf-of): Web application uses Oauth 2.0 auth code flow to acquire a token for a Web API. The web API validates the token, and calls Microsoft Graph on behalf of the user who authenticated in the web application.
 - [msal-node-extensions](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-extensions): Uses authorization code flow to acquire tokens then the [msal-extensions](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/msal-node-extensions) library to write the MSAL in-memory token cache to disk.
 
-## Build and Test
-
-- If you don't have [lerna](https://github.com/lerna/lerna) installed, run `npm install -g lerna`
-- Run `lerna bootstrap` from anywhere within `microsoft-authentication-library-for-js.git`.
-- Navigate to `microsoft-authentication-library-for-js/lib/msal-common` and run `npm run build`
-- Navigate to `microsoft-authentication-library-for-js/lib/msal-node` and run `npm run build`
-
-```javascript
-// to link msal-node and msal-common packages
-lerna bootstrap
-
-// Change to the msal-node package directory
-cd lib/msal-common/
-
-// To run build only for node package
-npm run build
-
-// Change to the msal-node package directory
-cd lib/msal-node/
-
-// To run build only for node package
-npm run build
-```
-
```

Comment:
This can be added back for now, I will update the contributing.md file and remove this in a future PR.